applications/ecallmgr/src/ecallmgr_config.erl

```diff
@@ -112,27 +116,27 @@ get_default(Key, Default) ->
 
 -spec get_json(kz_json:path()) -> api_object().
 -spec get_json(kz_json:path(), Default) -> kz_json:object() | Default.
--spec get_json(kz_json:path(), Default, kz_json:path()) -> kz_json:object() | Default.
+-spec get_json(kz_json:path(), Default, api_node_specific()) -> kz_json:object() | Default.
 get_json(Key) ->
-    get_json(Key, undefined).
+    get_json(Key, 'undefined').
 get_json(Key, Default) ->
     as_json_value(get(Key, Default), Default).
 get_json(Key, Default, Node) ->
     as_json_value(get(Key, Default, Node), Default).
 
 -spec as_json_value(any(), api_object()) -> api_object().
-as_json_value(undefined, Default) -> Default;
+as_json_value('undefined', Default) -> Default;
 as_json_value(V, Default) ->
     case kz_json:is_json_object(V) of
         'true' -> V;
         'false' -> Default
     end.
 
--spec get_jsons(kz_json:path()) -> api_objects().
+-spec get_jsons(kz_json:path()) -> kz_json:objects().
 -spec get_jsons(kz_json:path(), Default) -> kz_json:objects() | Default.
--spec get_jsons(kz_json:path(), Default, kz_json:path()) -> kz_json:objects() | Default.
+-spec get_jsons(kz_json:path(), Default, api_node_specific()) -> kz_json:objects() | Default.
 get_jsons(Key) ->
-    get_jsons(Key, 'undefined').
+    get_jsons(Key, []).
```

Comment:
Not sure what motivates this change?

Reply:
The only usages are in ecallmgr_fs_node and it was more convenient to return empty list and use the result, than to check for undefined.